compiler/record-minmax/CMakeLists.txt

```diff
@@ -15,6 +15,16 @@ target_link_libraries(record-minmax dio_hdf5)
 target_link_libraries(record-minmax vconone)
 target_link_libraries(record-minmax nncc_coverage)
 
+find_package(OpenMP QUIET)
+if (OpenMP_FOUND)
+  set(_OPENMP ON)
```

Comment:
```suggestion
  set(RECORDMINMAX_OPENMP ON)
```

Reply:
`_OPENMP` is generic and it hard to identify where this came from.

Reply:
Q) what is the purpose of `_OPENMP` set to `ON`?

Reply:
Actually, for clarity, but if it's unnecessary, I can remove :)

Reply:
OK, I think `_OPENMP` is defined from `OpenMP ` package. I you agree, we can remove this.